### ISSUE TITLE
Worldgen stalls when using LATITUDE on temperature_variation_scheme

### ISSUE THREAD

**Issue by vico93** [Glitchfiend/BiomesOPlenty#1594]
### What's the issue you encountered?

_Since nobody replied my questions in Discord, i feel i needed to make those here._

Since BoP got released for 1.15.2 i was attempting to create a world using [these settings on generation.toml](https://paste.dimdev.org/axigepewux.shell), which are similar to the options i've using back on 1.12. But everytime i tried to create a new world on singleplayer, it always got stuck on 100% and never let me join the game. I also tried to do it on a dedicated server and same thing: no players can join the server since it got stalled on 100% and didnt process to accept players.

### How can the issue be reproduced?

Basically set generation.toml with [this settings](https://paste.dimdev.org/axigepewux.shell).

### Logs

Watching the log window i cant see any worldgen error, the last line the log outputs when stall is always `preparing spawn area: 100%`. Nothing more appears and i need to kill java process using taskkill (Windows 10 x64).

### Mod Version
It goes back from the first builds of BoP for 1.15.2 and persists on `BiomesOPlenty-1.15.2-10.0.0.357-universal.jar`, Forge 31.1.74.
